xdsl/interactive/app.py

```diff
@@ -61,13 +62,22 @@ class InputApp(App[None]):
     input_text_area = TextArea(id="input")
     output_text_area = OutputTextArea(id="output")
 
+    list_of_passes = get_all_passes()
```

Comment:
this doesn't look right, if it's a constant, then it should be at the top level of the file, and probably a tuple instead of a string to enforce immutability